docusaurus.config.ts

```diff
@@ -0,0 +1,141 @@
+  // Set the /<baseUrl>/ pathname under which your site is served
+  // For GitHub pages deployment, it is often '/<projectName>/'
+  baseUrl: "/",
+
+  // GitHub pages deployment config.
+  // If you aren't using GitHub pages, you don't need these.
+  organizationName: "facebook", // Usually your GitHub org/user name.
+  projectName: "docusaurus", // Usually your repo name.
+
+  onBrokenLinks: "throw",
+  onBrokenMarkdownLinks: "warn",
+
+  // Even if you don't use internationalization, you can use this field to set
+  // useful metadata like html lang. For example, if your site is Chinese, you
+  // may want to replace "en" with "zh-Hans".
+  i18n: {
+    defaultLocale: "en",
+    locales: ["en"]
+  },
+
+  presets: [
+    [
+      "classic",
+      {
+        docs: {
+          sidebarPath: "./sidebars.ts",
+          // Please change this to your repo.
+          // Remove this to remove the "edit this page" links.
+          editUrl:
+            "https://github.com/facebook/docusaurus/tree/main/packages/create-docusaurus/templates/shared/"
+        },
+        blog: {
+          showReadingTime: true,
+          feedOptions: {
+            type: ["rss", "atom"],
+            xslt: true
+          },
+          // Please change this to your repo.
+          // Remove this to remove the "edit this page" links.
+          editUrl:
+            "https://github.com/facebook/docusaurus/tree/main/packages/create-docusaurus/templates/shared/",
+          // Useful options to enforce blogging best practices
+          onInlineTags: "warn",
+          onInlineAuthors: "warn",
+          onUntruncatedBlogPosts: "warn"
+        },
+        theme: {
+          customCss: "./src/css/custom.css"
+        }
+      } satisfies Preset.Options
+    ]
+  ],
+  plugins: [
+    [
+      require.resolve("docusaurus-lunr-search"),
+      {
+        // Optional: Specify options here
+        languages: ["en"] // Array of languages, default is ['en']
+      }
+    ]
+  ],
+  themeConfig: {
+    // Replace with your project's social card
+    image: "img/docusaurus-social-card.jpg",
+    navbar: {
+      logo: {
+        alt: "Mercurius Logo",
+        src: "images/mercurius-logo.svg"
+      },
+      items: [
+        { to: "/", label: "Home", position: "left" },
+        {
+          type: "docSidebar",
+          sidebarId: "customSidebar",
+          label: "Docs",
+          position: "left"
+        },
+        {
+          to: "/contribute",
+          label: "Contribute",
+          position: "left"
+        },
+        {
+          href: "https://github.com/mercurius-js/mercurius",
+          label: "GitHub",
+          position: "right"
+        }
+      ]
+    },
+    footer: {
+      style: "dark",
+      links: [
+        {
+          title: "Docs",
+          items: [
+            {
+              label: "Getting Started",
+              to: "/docs/guides/getting-started"
+            }
+          ]
+        },
+        {
+          title: "Community",
```

Comment:
It could be worth reference the mercurius discussions as well as SO?

Reply:
This is what this refers to: 

´´´
{
    title: “Community”,
    items: [
      {
        label: “Stack Overflow”,
        href="https://stackoverflow.com/questions/tagged/mercurius”
      }
    ]
  },
```

Reply:
Yeah I wonder if it's worth linking this as well? https://github.com/mercurius-js/mercurius/discussions

Reply:
If we want we can add it 🙂

Reply:
I have added the link to the github discussions for Mercurius, see here: https://github.com/mercurius-js/website/pull/1/commits/fb0735a0c2b42170bd98ee5afb8434ec14e7e6bf